File: hw/chavez/u3/hw17OPPConceptZoo/Python/OPPConceptsZoo/model/farm_animal.py
from datetime import date


class FarmAnimal:

    def __init__(self, id, breed, born_on, weight):
        self.id = id
        self.breed = breed
        self.born_on = born_on
        self.weight = weight

    def feed(self, food):
        print(f"{self.breed} is eating {food.description}")

File: hw/chavez/u3/hw17OPPConceptZoo/JavaScript/OPPConceptsZoo/model/FarmAnimal.js
export default class FarmAnimal {

    constructor(id, breed, bornOn, weight) {
        this.id = id;
        this.breed = breed;
        this.bornOn = bornOn;
        this.weight = weight;
    }

    feed(food) {
        console.log(
            `${this.breed} is eating ${food.description}`
        );
    }

}